include/dem/particle_particle_broad_search.h

```diff
@@ -78,18 +74,14 @@ class ParticleParticleBroadSearch
   void
   find_particle_particle_contact_pairs(
     dealii::Particles::ParticleHandler<dim> &particle_handler,
-    const std::vector<
-      std::vector<typename Triangulation<dim>::active_cell_iterator>>
-      *cells_local_neighbor_list,
-    const std::vector<
-      std::vector<typename Triangulation<dim>::active_cell_iterator>>
-      *cells_ghost_neighbor_list,
-    std::unordered_map<types::particle_index,
-                       std::vector<types::particle_index>>
-      &local_contact_pair_candidates,
-    std::unordered_map<types::particle_index,
-                       std::vector<types::particle_index>>
-      &ghost_contact_pair_candidates);
+    const typename dem_data_containers::dem_data_structures<
+      dim>::cells_neighbor_list *cells_local_neighbor_list,
+    const typename dem_data_containers::dem_data_structures<
+      dim>::cells_neighbor_list *cells_ghost_neighbor_list,
```

Comment:
Why are these pointers when the other ones were references?

Reply:
Corrected.